common/script/index.js

```diff
@@ -2279,13 +2279,14 @@ api.wrap = function(user, main) {
           user.stats.lvl++;
           tnl = api.tnl(user.stats.lvl);
           user.stats.hp = 50;
-          if (user.stats.lvl > api.maxLevel) {
+          var totalStatPoints = user.stats.str + user.stats.int + user.stats.con + user.stats.per;
+          if (user.stats.lvl > api.maxLevel && totalPoints >= 100 ) {
```

Comment:
100 should be api.maxLevel
I think the check for user.stats.lvl is no longer needed - checking total points should be enough?